### PR TITLE
Steven branch to dev 

### PR DESCRIPTION
**Pull request by Stevensauro**
Components are ready to use, fixed the structure again, my apologies on that!!
routes are properly configured now too

### CODE REVIEW

File: fetools-app/src/components/Header.jsx [hunk 1]
```diff
@@ -1,33 +1,119 @@
...
+                <svg id='hamburguer-btn' onClick={()=>setClick(!click)} width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg"
+                className="
+                cursor-pointer flex-2 fill-current self-center
+                sm:hidden
+                ">
+                    <path d="M4.875 27.625H34.125M4.875 19.5H34.125M4.875 11.375H34.125" stroke="black" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+                </svg>
+            )
+        } else{
+            return(
```
Comment (joekotvas) [inline fetools-app/src/components/Header.jsx:100]: Probably best to pull out these icons into their own files

File: fetools-app/src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        '>
+            <div className="
+            flex justify-between
+            sm:h-[39px]
+            ">
+                <h1 className="
+                flex-1 block min-w-fit max-w-fit font-bold text-[32px] leading-none
+                ">{appName}</h1>
+
+                <svg onClick={goTopPage} width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg"
```
Comment (joekotvas) [inline fetools-app/src/components/Footer.jsx:19]: Probably best to pull out these icons into their own files

File: fetools-app/src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { Link } from "react-router-dom"
+import { appName } from "../pages/data"
+
+export default function Footer(){
+    return(
+        <footer className='
```
Comment (joekotvas) [inline fetools-app/src/components/Footer.jsx:6]: Height here is intended to hug the contents, no need to specify.

File: fetools-app/src/components/Header.jsx [hunk 4]
```diff
@@ -1,33 +1,119 @@
...
+            sm:max-lg:flex-col sm:max-xl:h-[154px]  sm:max-md:p-4 sm:justify-center
+            md:max-lg:p-8
+            lg:items-center lg:h-[71px] lg:px-16 lg:gap-2
+            xl:px-16 xl:gap-16 min-[1440px]:gap-48
+            '>
+                <Link to="/" className="
+                            flex-1 block min-w-fit max-w-fit font-bold text-[32px] leading-none
+                            cursor-pointer
+                            sm:max-lg:self-center sm:max-lg:mb-4
+                            ">{appName}</Link>
```
Comment (joekotvas) [inline fetools-app/src/components/Header.jsx:23]: Good call abstracting out the app name as a variable.

File: fetools-app/src/components/Header.jsx [hunk 5]
```diff
@@ -1,33 +1,119 @@
...
+                <nav className='
```
Comment (joekotvas) [inline fetools-app/src/components/Header.jsx:24]: Consider pulling out the nav into a component here.

**Review by joekotvas — APPROVED** [review]

**Comment by Stevensauro** [comment]
I'll work on that boss!

**Pull request merged**